Meta/Lagom/CMakeLists.txt

```diff
@@ -49,14 +49,15 @@ file(GLOB LIBMARKDOWN_SOURCES "../../Libraries/LibMarkdown/*.cpp")
 file(GLOB LIBX86_SOURCES "../../Libraries/LibX86/*.cpp")
 file(GLOB LIBJS_SOURCES "../../Libraries/LibJS/*.cpp")
 file(GLOB LIBJS_SUBDIR_SOURCES "../../Libraries/LibJS/*/*.cpp")
+file(GLOB LIBCOMPRESS_SOURCES "../../Libraries/LibCompress/*.cpp")
```

Comment:
Oh nice, this builds in Lagom! :D Have a look at `test-crypto_lagom` a few lines down, can you do something like `test-compress_lagom`? This way, CI would automatically run the tests during build.